### PR TITLE
Synchronize files from linux-system-roles/template

### PR DESCRIPTION
**Pull request by systemroller**
This PR propagates files from [linux-system-roles/template](https://github.com/linux-system-roles/template) which should be in sync across [linux-system-roles](https://github.com/linux-system-roles) repos. In case of changing affected files via pushing to this PR, please do not forget also to push the changes to [linux-system-roles/template](https://github.com/linux-system-roles/template) repo.

CC: @i386x, @pcahyna.

### CODE REVIEW

**Comment by i386x** [comment]
Hi @dwlehman, besides `black`, I disable also `flake8` because its report also contains lot of issues (mainly about line lengths). Are you fine with this? If desirable, `flake8` can be enabled again.

**Comment by dwlehman** [comment]
Yes, it's fine with me to disable it for now.

**Comment by pcahyna** [comment]
@i386x please rebase it if needed.

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -8,6 +9,9 @@ lint:
   options:
     config-file: molecule/default/yamllint.yml
 platforms:
+  - name: centos-6
+    image: linuxsystemroles/centos-6
+    privileged: true
```
Comment (dwlehman) [inline molecule/default/molecule.yml]: This role does not support RHEL-6 at all, but I don't know if it should be listed as a platform or not.

**Comment by richm** [comment]
@i386x @pcahyna can one of you rebase this PR?

**Comment by richm** [comment]
I think this PR can be closed in favor of https://github.com/linux-system-roles/storage/pull/78

**Comment by richm** [comment]
closing in favor of #78

**Pull request closed without merging**